osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/metrics/MetricsService.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2022 Alliander N.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package org.opensmartgridplatform.shared.metrics;
+
+/** Interface to interact with a Metrics service */
+public interface MetricsService {
+
+  /**
+   * Increase the counter identified by {@code name} by 1.
+   *
+   * @param name Name of the counter
+   */
+  void increaseCounter(String name);
+
+  /**
+   * Increase the counter identified by {@code name} by 1.
+   *
+   * @param name Name of the counter
+   * @param description Description of the counter for humans
+   */
+  void increaseCounter(String name, String description);
```

Comment:
It feels a bit smelly that in this method it appears that on any invocation to influence the metric, a `description` is provided. I would expect the description to be related to the metric itself, and not to any specific increment of the counter. (What would it even mean if you were to call this for the same name, with different descriptions?)